Formula/arrayfire.rb

```diff
@@ -19,26 +19,29 @@ class Arrayfire < Formula
   depends_on "cmake" => :build
   depends_on "doxygen" => :build
   depends_on "fftw"
+  depends_on "fmt"
   depends_on "freeimage"
```

Comment:
No linkage. We should check whether this dependency is still needed.

Reply:
Still seems to be there:

https://github.com/arrayfire/arrayfire/blob/715e21fcd6e989793d01c5781908f221720e7d48/CMakeLists.txt#L56